docs/src/tutorial/cython_tutorial.rst

```diff
@@ -238,12 +301,20 @@ which we can try out in the interactive interpreter as follows::
 See, it works! And if you're curious about how much work Cython has saved you,
 take a look at the C code generated for this module.
 
-
 Cython has a way to visualise where interaction with Python objects and
 Python's C-API is taking place. For this, pass the
 ``annotate=True`` parameter to ``cythonize()``. It produces a HTML file. Let's see:
 
-.. figure:: htmlreport.png
+.. tabs::
+    .. group-tab:: Cython
+
+        .. figure:: htmlreport.png
+            :scale: 90 %
+
+    .. group-tab:: Pure Python
+
+        .. figure:: htmlreport_pure.png
+            :scale: 90 %
```

Comment:
Image (for pure) shows filenames from earlier revision

Reply:
Although this almost does not affect anything, I will redo it now.

Reply:
[here](https://github.com/cython/cython/pull/4226/commits/0614033ac1d1f776a2d4d5e698e12e42fc66ee88)